HW1/P4/P4_bfs.py

```diff
@@ -0,0 +1,28 @@
+def bfs(root, edge_lookup, sc): 
+	finished = sc.parallelize([(root, 0)])
+	new_size = sc.accumulator(0)
+
+	for i in range(10): 
+		new_size.value = 0 
```

Comment:
Why using "range(10)"? There are no such constraints as to limit the depth of our search to 10. Instead, you could use accumulators to record how many nodes you visited using each loop and terminates when accumulator equals to 0 meaning no nodes are found.